engine/resource/src/providers/provider_zip.cpp

```diff
@@ -74,52 +74,15 @@ static void DeleteZipArchiveInternal(dmResourceProvider::HArchiveInternal _archi
 
 static void CreateEntryMap(ZipProviderContext* archive)
 {
-    uint32_t count = archive->m_Manifest->m_DDFData->m_Resources.m_Count;
-    archive->m_EntryMap.SetCapacity(dmMath::Max(1U, (count*2)/3), count);
-
-    dmLiveUpdateDDF::HashAlgorithm algorithm = archive->m_Manifest->m_DDFData->m_Header.m_ResourceHashAlgorithm;
-    uint32_t hash_len = dmResource::HashLength(algorithm);
-
-    dmHashTable32<bool> added;
-    added.SetCapacity(dmMath::Max(1U, (count*2)/3), archive->m_EntryMap.Capacity());
-
     dmZip::HZip zip = archive->m_Zip;
-    for (uint32_t i = 0; i < count; ++i)
-    {
-        dmLiveUpdateDDF::ResourceEntry* entry = &archive->m_Manifest->m_DDFData->m_Resources.m_Data[i];
-
-        char hash_buffer[dmResourceArchive::MAX_HASH*2+1];
-        dmResource::BytesToHexString(entry->m_Hash.m_Data.m_Data, hash_len, hash_buffer, sizeof(hash_buffer));
-        hash_buffer[dmResourceArchive::MAX_HASH*2] = 0;
-
-        dmZip::Result zr = dmZip::OpenEntry(archive->m_Zip, hash_buffer);
```

Comment:
This guy uses `mz_zip_reader_locate_file` according to docs:
  
  >This function uses a simple linear search through the central
     directory, so it's not very fast.
  
https://github.com/richgel999/miniz/blob/9ae305f6e109f8f1fbd2130458f1ee6197269b3b/miniz.h#L58-L62